src/misc.jl

```diff
@@ -4,10 +4,11 @@
 #
 # See: https://en.wikipedia.org/wiki/Multivariate_gamma_function
 #
-function logmvgamma(p::Int, a::Float64)
-    res::Float64 = p * (p - 1.0) / 4.0 * log(pi)
+function logmvgamma(p::Int, a::Real)
+    # NOTE: one(a) factors are here for type stability for, e.g., Float32
```

Comment:
Instead of "type stability", I'd say something like "avoid unnecessary promotion of e.g.  `Float32`"